include/xtensor/xhistogram.hpp

```diff
@@ -138,10 +178,11 @@ namespace xt
 
         auto bin_edges = histogram_bin_edges(data, xt::ones<value_type>({ n }), bins);
```

Comment:
Should here be `auto&& bin_edges = ...` ?

The same applies to the next overload.